libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3372,6 +3372,12 @@ void GCS_MAVLINK::send_banner()
     if (hal.util->get_system_id(sysid)) {
         send_text(MAV_SEVERITY_INFO, "%s", sysid);
     }
+
+    // send RC output mode info if available
```

Comment:
I wonder if we can find a more appropriate place/time to emit this message.

I'm a bit worried we might end up losing chunks of information if we load too much into send_banner.

Reply:
I'm definitely open to suggestions on where it appears.  This nice thing about this location is it's done for all vehicles.

For copter a logical place would be at the end of Copter::init_rc_out().  Plane and Sub have similar functions but Rover and Tracker sadly do not.

Reply:
In a library would be better.  For the time being - it's fine as-is :-)